src/app/shared/components/clickable-item/clickable-item.component.html

```diff
@@ -2,7 +2,7 @@
   'switcher-card-item': isSwitcherCard,
   'active': active
 }"  aria-label="clickable item" >
-  <ion-button class="accessible-btn" aria-label="switcher card" role="button">
+  <ion-button class="accessible-btn" [ngClass]="{'accessible-item': isCustomizedCard, 'accessible-chat': isChatCard}" aria-label="switcher card" role="button">
```

Comment:
`accessible-chat` css class is an empty css class, it didn't have any styles in it. so why we need extra logic to check `isChatCard` and add that class.

Reply:
@sasangachathumal , it will be fixed in another story. :D sure I'll remove it here. thanks